tests/QARegressionTests/test_Issues/test_CoreIssue165.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import pandas as pd
+from business_rules.operators import DataframeType
+import pytest
+
+"""This automated test only validates the acceptance criteria for core issue 165
+for complete test cases coverage, refer to unit tests"""
+
+
+@pytest.mark.regression
+class TestCoreIssue165(unittest.TestCase):

```

Comment:
These tests are covered in the unit tests

Reply:
removed